src/Libraries/Microsoft.Extensions.Telemetry.Abstractions/Latency/Registration/LatencyRegistryExtensions.cs

```diff
@@ -79,7 +80,7 @@ private static void CheckNames(string[] names)
 
     private static void ConfigureOption(this IServiceCollection services, Action<LatencyContextRegistrationOptions> action)
     {
-        _ = services.AddOptions<LatencyContextRegistrationOptions>();
+        _ = services.AddValidatedOptions<LatencyContextRegistrationOptions, LatencyContextRegistrationOptionsValidator>();
```

Comment:
Do we need the options type to be public at all?

Reply:
I asked Ananth for feedback on that. Maybe we should hide it, we'll see what he ssys.

Reply:
He's OOF, so it'll take a while. We can address this in a separate PR.